app/views/mentee_profiles/dashboard.html.slim

```diff
@@ -1,6 +1,7 @@
 .col-xs-2.col-md-3.col-lg-2
-  h3.mentee-name Jane Doe
-  = image_tag "profile/mentee_icon.jpg", alt: "Mentee profile picture", class: "profile-icon"
+  h3.mentee-name 
+    = current_user.first_name
+  = image_tag current_user.avatar_url(:thumb) , alt: "Mentee profile picture", class: "profile-icon" if current_user.avatar?
```

Comment:
yes it is :) we'll still need to add some default img if user has no avatar but i'll be adding something like that for organizer so i'm gonna add it here later